source/user-manual/api/reference.rst

```diff
@@ -6848,311 +7198,8 @@ Returns the sca checks of an agent
-	            "status": "Not applicable",
-	            "reason": "Directory /var/www not found",
-	            "policy_id": "system_audit",
-	            "id": 1004,
-	            "result": "",
-	            "compliance": [
-	               {
-	                  "key": "pci_dss",
-	                  "value": "6.5, 6.6, 11.4"
-	               }
-	            ],
-	            "rules": [
-	               {
-	                  "type": "directory",
-	                  "rule": "d:$web_dirs -> ^.yop$;"
-	               }
-	            ]
-	         },
-	         {
-	            "title": "PHP - Displaying of errors is enabled",
-	            "status": "Not applicable",
-	            "file": "/etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini",
-	            "reason": "File /etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini not found",
-	            "policy_id": "system_audit",
-	            "id": 1003,
-	            "result": "",
-	            "rules": [
-	               {
-	                  "type": "file",
-	                  "rule": "f:$php.ini -> r:^display_errors = On;"
-	               }
-	            ]
-	         },
-	         {
-	            "title": "PHP - Allow URL fopen is enabled",
-	            "status": "Not applicable",
-	            "file": "/etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini",
-	            "reason": "File /etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini not found",
-	            "policy_id": "system_audit",
-	            "id": 1002,
-	            "result": "",
-	            "rules": [
-	               {
-	                  "type": "file",
-	                  "rule": "f:$php.ini -> r:^allow_url_fopen = On;"
-	               }
-	            ]
-	         },
-	         {
-	            "title": "PHP - Expose PHP is enabled",
-	            "status": "Not applicable",
-	            "file": "/etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini",
-	            "reason": "File /etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini not found",
-	            "policy_id": "system_audit",
-	            "id": 1001,
-	            "result": "",
-	            "rules": [
-	               {
-	                  "type": "file",
-	                  "rule": "f:$php.ini -> r:^expose_php = On;"
-	               }
-	            ]
-	         },
-	         {
-	            "title": "PHP - Register globals are enabled",
-	            "status": "Not applicable",
-	            "file": "/etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini",
-	            "reason": "File /etc/php.ini,/var/www/conf/php.ini,/etc/php5/apache2/php.ini not found",
-	            "policy_id": "system_audit",
-	            "id": 1000,
-	            "result": "",
-	            "rules": [
-	               {
-	                  "type": "file",
-	                  "rule": "f:$php.ini -> r:^register_globals = On;"
-	               }
-	            ]
-	         }
-	      ]
+	      "totalItems": 0,
```

Comment:
SCA checks should appear, otherwise this is not a good example.

Reply:
Done!